tests/Functional/ContentOrganisationRenderTest.php

```diff
@@ -30,6 +31,8 @@ class ContentOrganisationRenderTest extends BrowserTestBase {
     'node',
     'address',
     'oe_media',
+    'media_avportal',
+    'oe_media_avportal',
```

Comment:
One or both of these two modules should be removed from this test and listed as dependencies of the oe_theme_content_org module. Also update the readme.

Reply:
done in oe_content https://github.com/openeuropa/oe_content/blob/EWPP-323/modules/oe_content_organisation/oe_content_organisation.info.yml#L14